docs/pmm-upgrade/upgrade_agent.md

```diff
@@ -1,10 +1,50 @@
-# Upgrade PMM agent
+# Upgrade PMM Client
 
-PMM-Agent can be updated from tarball:
-{.power-number}
+There are two primary methods to update PMM Clients, depending on your initial installation method:
 
- 1. Download `tar.gz` with `pmm2-client`.
- 2. Extract it.
+1. Using your operating system's package manager.
+2. Updating from a tarball.
+
+### 1. Package Manager method
+
+The package manager method is generally more convenient and efficient. Percona provides the [percona-release](https://docs.percona.com/percona-software-repositories/installing.html) package, which helps you install Percona software, including the PMM Client. The PMM Client is available from the `pmm-client` repository.
+
+To deploy a new version of the Client via package manager, simply replace the currently installed package with the latest version of the PMM Client or with a specific version.
+
+#### Install the latest PMM Client version
+
+Run the commands below to install the latest PMM Client version via package manager and keep your existing Client configuration during the update process.
+
+For example, to install the latest version of the PMM Client on Red Hat or its derivatives:
+
+ ```sh
+ percona-release enable pmm-client
+ yum update pmm-client
+ ```
+
+#### Deploy a specific version
+
+To deploy a specific version of the PMM Client via package manager, check the available versions and then provide the full name of the package. For example:
+
+ ```sh
+ yum --showduplicates search pmm2-client
+ pmm2-client-2.39.0-6.el9.x86_64 : Percona Monitoring and Management Client (pmm-agent)
+ pmm2-client-2.40.0-6.el9.x86_64 : Percona Monitoring and Management Client (pmm-agent)
+ pmm2-client-2.40.1-6.el9.x86_64 : Percona Monitoring and Management Client (pmm-agent)
+ pmm2-client-2.41.0-6.el9.x86_64 : Percona Monitoring and Management Client (pmm-agent)
+ pmm2-client-2.41.1-6.el9.x86_64 : Percona Monitoring and Management Client (pmm-agent)
+ pmm2-client-2.41.2-6.el9.x86_64 : Percona Monitoring and Management Client (pmm-agent)
+ pmm2-client-2.42.0-6.el9.x86_64 : Percona Monitoring and Management Client (pmm-agent)
+
+ yum update pmm2-client-2.41.2-6.el9.x86_64
```

Comment:
```suggestion
 ```sh
 yum --showduplicates search pmm-client
 pmm-client-3.0.0-6.el9.x86_64 : Percona Monitoring and Management Client (pmm-agent)
 pmm-client-3.0.1-6.el9.x86_64 : Percona Monitoring and Management Client (pmm-agent)

 yum update pmm-client-3.0.1-6.el9.x86_64
```

Reply:
Note: the packages do not exist yet, but they are anticipated to be like that.